Core.IntegrationTests/GitBackedTestBase.cs

```diff
@@ -96,15 +97,34 @@ protected static string ExecuteGitCommandWithOutput (string argument)
     psi.RedirectStandardOutput = true;
     psi.RedirectStandardError = true;
 
-    using var command = Process.Start(psi);
-    command.WaitForExit();
-    if (command.ExitCode != 0)
+    var process = new Process
+    {
+        StartInfo = psi
+    };
+    
+    var errorBuilder = new StringBuilder();
+    process.ErrorDataReceived += new DataReceivedEventHandler((sender, e) =>
+    {
+      errorBuilder.Append(e.Data);
+    });
```

Comment:
We can simplify this here to use a lambda `process.ErrorDataReceived += (sender, e) => errorBuilder.Append(e.Data);`